Description: that file needs to be used on windows only.

The build can also be fixed by exporting
VBOX_WITH_HOST_SHIPPING_AUDIO_TEST=
VBOX_WITH_VALIDATIONKIT=

in LocalConfig.kmk

diff --git a/src/VBox/ValidationKit/Config.kmk b/src/VBox/ValidationKit/Config.kmk
index 7047a04b8..66ea42ca4 100644
--- a/src/VBox/ValidationKit/Config.kmk
+++ b/src/VBox/ValidationKit/Config.kmk
@@ -122,7 +122,7 @@ TEMPLATE_VBoxValidationKitR3_LNK_DEPS.win.amd64  = $(if $(eq $(tool_do),LINK_LIB
 
 #TODO: TEMPLATE_VBoxValidationKitR3_EXTENDS = VBOXGUESTR3EXE
 
-TEMPLATE_VBoxValidationKitR3_USES += vboximportchecker
+TEMPLATE_VBoxValidationKitR3_USES.win += vboximportchecker
 TEMPLATE_VBoxValidationKitR3_VBOX_IMPORT_CHECKER.win.x86   = nt31
 TEMPLATE_VBoxValidationKitR3_VBOX_IMPORT_CHECKER.win.amd64 = xp64
 
